### PR TITLE
Remove stuff from preview pane

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Fixes a part of #2430 

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [ ] ~Was the documentation framework updated to reflect the changes?~
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by checking the preview pane in the UI
* [x] If applicable, add screenshots or log transcripts of the feature working

image

### CODE REVIEW

**Review by ilija-lazoroski — APPROVED** [review]
LGTM!

File: monkey/monkey_island/cc/services/node.py [hunk 1]
```diff
@@ -41,33 +41,20 @@ def get_displayed_node_by_id(node_id, for_report=False):
             new_node["ip_addresses"] = node["ip_addresses"]
             new_node["domain_name"] = node["domain_name"]
 
-        accessible_from_nodes = []
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/services/node.py:44]: This whole file will go away

Comment (shreyamalviya) [reply monkey/monkey_island/cc/services/node.py:44]: In #2273, yes. I don't think we can remove it right now.

**Review by VakarisZ — APPROVED** [review]
Approved, but the changelog could be rephrased

File: CHANGELOG.md [hunk 2]
```diff
@@ -121,6 +121,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - "GET /api/test/monkey" endpoint. #2269
 - "GET /api/test/log" endpoint. #2269
 - Node Map from Security Report. #2334
+- "Accessible From" and "Services" from the preview pane in the map on the Island. #2430
```
Comment (VakarisZ) [inline CHANGELOG.md:124]: Island doesn't say much to a user unfamiliar with our architecture. Instead we should say something like "from the preview pane in the map page". Page means UI, it's clear what's been done

**Pull request merged**